packages/pds-ember/addon/components/pds/textarea/index.hbs

```diff
@@ -0,0 +1,14 @@
+<textarea
+  class="
+    pds-textarea
+    {{if @invalid 'pds--invalid'}}
+  "
+  rows="4"
```

Comment:
Does this just save everyone from having to specify height in `em` or are there other a11y reasons for using `rows`?

Reply:
TL;DR - The `rows` attr is simpler for modifying its appearance.

Browser rendering logic for a `<textarea>` will calculate the _content_ box height based on the `rows` attr (taking into account `font-size` and `line-height` styles), but it will also calculate the _border_ height based on the calculated content box (taking into account `padding` and `border` styles).

Additionally, for UX reasons, the bottom padding of the textarea has been tweaked so that the bottom edge of the element only covers about 1/2 of a line of text (should content have more lines than defined by `[rows]`), which provides a visual affordance to users that there may be more content than displayed.

Using the `[rows]` attr is easier than requiring consumers to calculate all of this the math on their own.